bbb/services.py

```diff
@@ -160,7 +160,16 @@ def _handleFinishedRequest(self, brid, properties, results):
             log.debug("Task not found for brid %s, nothing to do.", brid)
             return
 
-        log.debug("brid %i : taskId %s : runId %i", brid, taskid, runid)
+        log.info("Handling finished task %s run %s brid %s", taskid, runid,
+                 brid)
+        # Try to claim the task in case if the "started" event comes after
+        try:
+            self.tc_queue.claimTask(taskid, runid, {
+                "workerGroup": self.tc_worker_group,
+                "workerId": self.tc_worker_id,
+            })
+        except TaskclusterRestFailure:
+            log.info("Cannot claim task %s run %s, assuming it is claimed already", taskid, runid)
```

Comment:
My only concern here is that the logging is going to be pretty noisy. In the normal case we're going to have a lot of "Cannot claim task ..." messages, which kind of sounds like an error. Maybe turn this into a log.debug() call, and have a log.info() call if we successfully claimed the task above?